README.md

```diff
@@ -448,6 +448,18 @@ config :nerves_hub_link, remote_iex_timeout: 900 # 15 minutes
 You may also need additional permissions on NervesHub to see the device and to use the
 remote IEx feature.
 
+## CA Certificates
+
+NervesHubLink allows configuring a custom CA Certificate store. In your `config.exs`:
+
+```elixir
+# CA Certs for the Device/Server connection
+config :nerves_hub_link, ca_store: :nerves_hub_ca_store
+
+# CA Certs for the Host/API connection
+config :nerves_hub_cli, ca_store: :nerves_hub_ca_store
```

Comment:
```suggestion
config :nerves_hub_user_api, ca_store: :nerves_hub_ca_store
```

`NervesHubCli` doesn't have a concept of the CA/request structure. Should this be `:nerves_hub_user_api`?